src/main/java/mate/bookshopapp/dto/book/BookDto.java

```diff
@@ -1,12 +1,17 @@
 package mate.bookshopapp.dto.book;
 
 import java.math.BigDecimal;
+import java.util.Set;
+import lombok.Data;
 
-public record BookDto(Long id,
-                      String title,
-                      String author,
-                      String isbn,
-                      BigDecimal price,
-                      String description,
-                      String coverImage) {
+@Data
+public class BookDto {
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+    private Set<Long> categoryIds;
 }
```

Comment:
Having both records and base models with @Data annotation as DTOs is confusing.
Please unify approach and use same everywhere